models/base/manifest/snowplow_media_player_base_sessions_lifecycle_manifest.sql

```diff
@@ -0,0 +1,96 @@
+{{
+  config(
+    materialized="incremental",
+    unique_key='session_id',
+    upsert_date_key='start_tstamp',
+    sql_header=snowplow_utils.set_query_tag(var('snowplow__query_tag', 'snowplow_dbt')),
+    partition_by = snowplow_utils.get_value_by_target_type(bigquery_val={
+      "field": "start_tstamp",
+      "data_type": "timestamp"
+    }, databricks_val='start_tstamp_date'),
+    full_refresh=snowplow_media_player.allow_refresh(),
+    tags=["manifest"],
+    snowplow_optimize=true
+  )
+}}
+
+-- Known edge cases:
+-- 1: Rare case with multiple domain_userid per session.
+
+{% set lower_limit, upper_limit, _ = snowplow_utils.return_base_new_event_limits(ref('snowplow_media_player_base_new_event_limits')) %}
+{% set session_lookback_limit = snowplow_utils.get_session_lookback_limit(lower_limit) %}
+{% set is_run_with_new_events = snowplow_utils.is_run_with_new_events('snowplow_media_player') %}
+
+with new_events_session_ids as (
+  select
+    e.domain_sessionid as session_id,
+    max(e.domain_userid) as domain_userid, -- Edge case 1: Arbitary selection to avoid window function like first_value.
```

Comment:
Just checking currently you haven't added support for mobile events right? Just adding a base so that you _can_?

Reply:
Right, just adding a base so that I can later add support for mobile. I don't have a mobile event dataset with the old media schemas, so need to first add support for new schemas and then add support for mobile to be able to test it.